lib/serializer.js

```diff
@@ -2,13 +2,17 @@
 
 // external modules
 var assert = require('assert-plus');
+var EventEmitter = require('events');
 var _ = require('lodash');
 var nerror = require('@netflix/nerror');
-var domain = require('domain');
 var MultiError = nerror.MultiError;
 var VError = nerror.VError;
 var safeJsonStringify;
 
+// We load the domain module lazily to avoid performance regression on Node.js
```

Comment:
Just stumbled on a previous comment at https://github.com/restify/errors/pull/95#discussion_r261760316, I'm glad we're addressing this now 🙏